.github/workflows/arbitrator-ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           cd go-ethereum
           # Enable KZG point evaluation precompile early
           sed -i 's#var PrecompiledContractsBerlin = map\[common.Address\]PrecompiledContract{#\0 common.BytesToAddress([]byte{0x0a}): \&kzgPointEvaluation{},#g' core/vm/contracts.go
-          go build -o /usr/local/bin/geth ./cmd/geth
+          go build -o /usr/local/bin/geth -ldflags='-checklinkname=0' ./cmd/geth
```

Comment:
Let's just update the pinned geth version above from v1.13.8 to the latest release. We might not even need the patch applied above anymore to enable the KZG evaluation precompile.